### DISCUSSION TITLE
Is it possible to add a fade in/out effect when navigating between images?

### DISCUSSION THREAD

**Discussion by faustonunez** [igordanchenko/yet-another-react-lightbox#229] [Q&A] [answered]
At the moment, when you navigate between images, the images slide, but is it possible to add other effects? For instance, make the images fade in/out when clicking back and next

**Comment by igordanchenko** [comment, marked as answer]
Please see my previous response to this question in #149

**Reply by faustonunez** [reply]
I see! Thank you for the quick response.

**Reply by huantran123** [reply]
Hi, I’m not able to access the CodeSandbox attached in that discussion. Could you please send a StackBlitz for it? Thank you!

**Reply by igordanchenko** [reply]
I've updated my [answer](https://github.com/igordanchenko/yet-another-react-lightbox/discussions/149#discussioncomment-6379276) on that thread with CSS styles from the no longer functioning CodeSandbox example. I hope this helps.
